pandas/conftest.py

```diff
@@ -278,6 +278,18 @@ def nulls_fixture(request):
 nulls_fixture2 = nulls_fixture  # Generate cartesian product of nulls_fixture
 
 
+@pytest.fixture(params=[None, np.nan, pd.NaT])
+def unique_nulls_fixture(request):
+    """
```

Comment:
can't you use the existing nulls_fixture? (ahh or that has the float NaN which *are* the same as np.nan)?

Reply:
yes this is ok (i see how you are using it below).